master/internal/stream/util.go

```diff
@@ -102,21 +103,21 @@ func getStreamableScopes(accessMap map[model.AccessScopeID]bool) (bool, []model.
 func processQuery(
 	ctx context.Context,
 	createFilteredIDQuery func() *bun.SelectQuery,
-	since int64, known string,
+	since int64, known string, seq string,
 ) (string, []int64, error) {
 	// step 1: calculate all ids matching this subscription
 	oldEventsQuery := createFilteredIDQuery()
 	newEventsQuery := createFilteredIDQuery()
 	// get events that happened prior to since that are relevant (appearance)
-	oldEventsQuery.Where("seq <= ?", since)
+	oldEventsQuery.Where(fmt.Sprintf("%s <= ?", seq), since)
 	var exist []int64
 	err := oldEventsQuery.Scan(ctx, &exist)
 	if err != nil && errors.Cause(err) != sql.ErrNoRows {
 		log.Errorf("error when scanning for old offline events: %v\n", err)
 		return "", nil, err
 	}
 	// and events that happened since the last time this streamer checked
-	newEventsQuery.Where("seq > ?", since)
+	newEventsQuery.Where(fmt.Sprintf("%s > ?", seq), since)
```

Comment:
Do we anticipate the column name for the sequence number to change? Would it not make sense to either define this as a constant on either a per-streamable type basis or here in utils.go (if we are assuming that the name of the sequence number column will be consistent across all tables)

Reply:
We are joining `models` and `model_versions` to get the workspace id, so `seq` ends up ambiguous

Reply:
Would it make sense to use something like `entityTableAlias` instead of `seq`? 

I'm concerned that it might not be intuitive to understand the purpose of that parameter without this context.

Reply:
So instead of:
```
newEventsQuery.Where(fmt.Sprintf("%s > ?", seq), since)
```
we could do something like:
```
newEventsQuery.Where(fmt.Sprintf("%s.seq > ?", entityTableAlias), since)
```

Reply:
Sure, updated!